mne/epochs.py

```diff
@@ -1556,13 +1556,10 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             If 'truncate', events will be truncated from the end of each event
             list. If 'mintime', timing differences between each event list
             will be minimized.
-        copy : bool
-            If True, a copy of epochs will be returned. Otherwise, the
-            function will operate in-place.
```

Comment:
I agree on that. Always modifying `self` might not be desired by everyone. But then again, I feel it would be convenient to always work on `self`. Perhaps it would be a good idea to change the default value of  `copy` to `False`. What do you think?